Zend/zend_language_scanner.l

```diff
@@ -631,6 +634,27 @@ zend_op_array *compile_filename(int type, zval *filename)
 		convert_to_string(&tmp);
 		filename = &tmp;
 	}
+	if (!CG(script_extensions)[0]) {
```

Comment:
@yohgaki what you mean by "works as supposed"? Would the filename above be allowed? If yes, this protection is not very useful - I still can inject any code with vulnerable include.